tests/run/cpp_stl_forward_list.pyx

```diff
@@ -0,0 +1,61 @@
+# mode: run
+# tag: cpp, werror, cpp11
+# distutils: extra_compile_args=-std=c++11
```

Comment:
This is added automatically by `runtests.py` when the `cpp11` tag is used.

Reply:
@scoder I can remove this then. I was just basing it off of old files I had seen. Thanks for the information.